lib/open_project/authentication/manager.rb

```diff
@@ -0,0 +1,42 @@
+module OpenProject
+  module Authentication
+    class Manager < Warden::Manager
+      serialize_into_session do |user|
```

Comment:
I don't agree either. With common use cases such as `#map` I agree. But not here even though it is essentially just a map too. With the short form it's not obvious what is being mapped, however.